### DISCUSSION TITLE
New context menu on Win11 is still disabled after I deleted EP

### DISCUSSION THREAD

**Discussion by acerola0rion598** [valinet/ExplorerPatcher#1796] [Q&A] [answered]
It's not like I am unhappy with this, but I want to know is it a toggleable feature somewhere in settings or politics?

**Comment by pyrates999** [comment]
Can you show a picture of what you mean by the new context menu?

**Reply by acerola0rion598** [reply]
This one
![image](https://github.com/valinet/ExplorerPatcher/assets/60043710/a34a3a7d-dffd-46ab-8244-5b41f0c54c40)

**Comment by pyrates999** [comment, marked as answer]
This might be the answer: https://pureinfotech.com/bring-back-classic-context-menu-windows-11/
